t/03-testapi.t

```diff
@@ -608,6 +608,32 @@ subtest 'compat_args' => sub {
     like(warning { testapi::compat_args(\%def_args, [], 'outch') }->[0], qr/^Odd number of arguments/, 'Warned on Odd number 2');
 };
 
+subtest 'check_quiet_option' => sub {
```

Comment:
no need to escape any whitespace within the string. You can use something more explanatory here if you like

Reply:
ok